packages/web-components/src/components/va-telephone/test/va-telephone.e2e.ts

```diff
@@ -132,6 +132,18 @@ describe('va-telephone', () => {
     expect(link.innerText).toEqual('711');
   });
 
+  it('handles a vanity number as a contact prop', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<va-telephone contact="877222VETS"></va-telephone>');
+
+    const link = await page.find('va-telephone >>> a');
+    expect(link.getAttribute('aria-label')).toEqual(
+      '1. 8 7 7. 2 2 2. V E T S. 8 3 8 7.',
+    );
```

Comment:
I don't think the current implementation puts the vanity letters into the aria-label. Is this something we want? cc @CourtneyChambers77 

https://github.com/department-of-veterans-affairs/component-library/blob/41015c4b2de355949e03ab0c450fc8e7535804b4/packages/react-components/src/components/Telephone/Telephone.jsx#L135-L139

https://github.com/department-of-veterans-affairs/component-library/blob/41015c4b2de355949e03ab0c450fc8e7535804b4/packages/react-components/src/components/Telephone/Telephone.jsx#L53-L65

![image](https://user-images.githubusercontent.com/2008881/157323663-7c8e72d2-d6a6-4658-a4e4-05d4e3f4aa5b.png)

Reply:
I have updated the PR so that the aria-label matches the image above and removes the extra "VETS"